web-ui/src/main/resources/catalog/components/utility/UtilityService.js

```diff
@@ -380,8 +380,14 @@
               var values = angular.isArray(prop.if[key]) ? prop.if[key] : [prop.if[key]];
 
               var recordValue = this.getObjectValueByPath(record, key);
-              if (values.includes(recordValue)) {
-                cb();
+              if (angular.isArray(recordValue)) {
+                if (values.filter((value) => recordValue.includes(value)).length > 0) {
```

Comment:
Produces the following error in the log file:

```
SEVERE: /catalog/components/utility/UtilityService.js:384: ERROR - this language feature is only supported in es6 mode: short function syntax. Use --language_in=ECMASCRIPT6 or ECMASCRIPT6_STRICT to enable ES6 features.
                if (values.filter((value) => recordValue.includes(value)).length > 0) {
```

To check to use Lodash.

Reply:
Fixed. Can be done with JS.